CMakeLists.txt

```diff
@@ -427,7 +427,13 @@ if (NOT MSVC AND NOT APPLE AND NOT ANDROID)
     find_library(NUMA NAMES NUMA libnuma.so.1)
     if (NUMA)
         message(STATUS "Found libnuma: ${NUMA}")
-        list(APPEND QUIC_COMMON_DEFINES CXPLAT_NUMA_AWARE)
+        find_file(NUMA-DEV NAMES "numa.h")
```

Comment:
I think the standard way is to try to compile code fragment. But this will probably work as well on moth systems.